assets/javascripts/discourse/templates/admin-wizards-transfer.hbs

```diff
@@ -0,0 +1,34 @@
+<h3>Select Wizards to export</h3>
+{{log this}}
+
+<form {{action 'export' wizards on="submit"}} class="form">
+  <ul>
+    {{#each model as |w|}}
+
+      <li style="list-style-type: none;">
```

Comment:
We typically don't put styles in the html like this. Pop this in the stylesheet for this plugin.

Reply:
@fzngagan We've still got styles in the html here.